src/core/power_flow_data_wrapper.jl

```diff
@@ -16,10 +16,11 @@ check_network_reduction(::PFS.SystemPowerFlowContainer) = nothing
 
 function check_network_reduction(pfd::PFS.PowerFlowData)
     nrd = PFS.get_network_reduction_data(pfd)
-    if length(PNM.get_reverse_bus_search_map(nrd)) != 0
+    if !isempty(PNM.get_reductions(nrd))
         throw(
             IS.NotImplementedError(
-                "Network reduction is not supported in PowerSimulations.jl.",
+                "Network reductions of types $(PNM.get_reductions(nrd)) are not supported " *
+                "in PowerSimulations.jl.",
```

Comment:
This comment is misleading: network reductions are supported in PowerSimulations.jl, just not yet for power flow in the loop.